jobs.html

```diff
@@ -144,6 +144,18 @@ <h3 id="student-assistant-systems-administrator---morgridge"><a href="https://mo
             </p>
             <p><a href="https://morgridge.org/job-posting/student-assistant-systems-administrator/">View Job Posting</a></p>
 
+            <h3 href="https://iris-hep.org/fellows.html">IRIS-HEP Fellowship</h3>
+            <p>
+                IRIS-HEP is accepting proposals for Summer 2021 Fellowship Program from undergraduate
+                students and graduate students that do not have a research assistantship (e.g. Masters students).
+                Known for discovering the Higgs Boson, IRIS-HEP, partners with CHTC to help advance
+                the state of distributed high throughput computing. The
+                <a href="https://iris-hep.org/fellow_projects.html">fellowship proposals</a> that we
+                are associated with pertain to Kubernetes and containers and will have Brian
+                Lin listed as a contact.
+            </p>
+            <p><a href="https://iris-hep.org/fellows.html">Learn more about the Fellowship</a></p>
```

Comment:
```suggestion
            <p><a href="https://iris-hep.org/fellows.html">Learn more about the IRIS-HEP Fellowship Program</a></p>
```

As much as I like the LOTR reference, we should probably be a little clearer here